### PR TITLE
Update jupyter-support-py.md

### PR DESCRIPTION
**Pull request by hoangmt**
Small change on the section Connect to a remote Jupyter server.

### CODE REVIEW

File: docs/python/jupyter-support-py.md [hunk 1]
```diff
@@ -105,6 +105,9 @@ You can offload intensive computation in a Jupyter notebook to other computers b
 
 To connect to a remote Jupyter server:
 
+0.a. Open vs code, make sure you have not opened any ipynb yet before proceeding.
```
Comment (rchiodo) [inline docs/python/jupyter-support-py.md]: Not sure the open vs code suggestion is a good one. It's better to just start the jupyter server before you even start VS code.

I think I'd reword this as:
```
Start jupyter notebook or jupyter labs before running VS Code. 
Copy the URL presented under the text 'Or copy and paste one of these URLs'. They should have a form similar to: 
http://localhost:8889/?token=d9539e71c86b55da970823b15cf1ddafa14e003e1f67a42d
```

Comment (rchiodo) [reply docs/python/jupyter-support-py.md]: Or maybe even more explicit:
```
Start 'jupyter notebook' or 'jupyter lab' before opening any .ipynb files inside of VS code.
```

Comment (hoangmt) [reply docs/python/jupyter-support-py.md]: This is a very good suggestion.

**Comment by gregvanl** [comment]
Hi @hoangmt Are you going to update your PR with the suggestions from @rchiodo?

**Comment by gregvanl** [comment]
Closing as haven't heard back from PR author.
